package.json

```diff
@@ -53,8 +53,10 @@
         "@rocket.chat/apps-engine": "^1.22.2",
         "figures": "^3.0.0",
         "fs-extra": "^8.1.0",
+        "fs-monkey": "github:RocketChat/fs-monkey",
```

Comment:
Here we use our forked [fs-monkey](https://github.com/RocketChat/fs-monkey/commit/ef7170839caaa34fa079d9f4e88ba1efdd5a6ee0) version, since we'll need an [unpatch](https://github.com/RocketChat/Rocket.Chat.Apps-compiler/pull/30/files#diff-6aa0c4f0a29a6e953a8ddb67e9041280d1185b4cf9d9f3ed183298d66f5db504R83) function to get `require`'s behaviour back to normal.

Reply:
Adding this dependency and the `memfs`, along with the `MemFS` class, seems to add quite a bit of complexity to the project IMO :/

I'm wondering if we could provide a custom `require` for that instead, based on the one we have here on the Apps-Engine https://github.com/RocketChat/Rocket.Chat.Apps-engine/blob/master/src/server/misc/Utilities.ts#L45

Do you think that would work?

Reply:
Thanks for the information! It helped me a lot ;)